cmd/config.go

```diff
@@ -117,13 +117,14 @@ func newConfigRmCmd(stack *string) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			stackName := s.Ref().Name()
```

Comment:
dumb question.   why is Name no longer a property of the stack, but now something off of this 'Ref' guy?